DataConnectors/GoogleWorkspaceReports/GWorkspaceReportsAPISentinelConnector/__init__.py

```diff
@@ -128,19 +128,25 @@ def post_data(customer_id, shared_key, body, log_type):
         logging.warn("Response code: {}".format(response.status_code))
 
 def expand_data(obj):
+    new_obj = []
     for event in obj:
-        for nested in event["events"]:
-            if 'name' in nested:
-                event.update({'event_name': nested["name"]})
-            if 'type' in nested:
-                event.update({'event_type': nested["type"]})
-            if 'parameters' in nested:
-                for parameter in nested["parameters"]:
-                    if 'name' in parameter:
-                        for param_name in ["value", "boolValue", "multiValue", "multiMessageValue", "multiIntValue", "messageValue", "intValue"]:
-                            if param_name in parameter:
-                                event.update({parameter["name"]: parameter[param_name]})
-    return obj
+        nested_events_arr = event["events"]

```

Comment:
@averbn, I have tested with provided [sample data ](https://github.com/Azure/Azure-Sentinel/blob/master/Sample%20Data/Custom/GWorkspace_ReportsAPI_drive_CL.json) which contains array of events but it sends only some of the events to Sentinel.

Can you please check with array of events sample data ? Thanks!

Reply:
@v-maudan , I updated sample data according to the last changes with [events] expanding, pls check

Reply:
@averbn , Current logic is still not working for nested events. Can you please check and confirm ? 
As per below mention scenrio it will only ingest event 3 data.
...
"items": [
  { // item 1
   ...
   "events": [
    { //event 1
    },
    { //event 2
    },
    { //event 3
    },
  { // item 2
...`
`

Reply:
@averbn , Thanks for fix. I have verified with nested sample data and it's working fine.